Formula/cacli.rb

```diff
@@ -2,7 +2,7 @@ class Cacli < Formula
   desc "Train machine learning models from Cloud Annotations"
   homepage "https://cloud.annotations.ai"
   url "https://github.com/cloud-annotations/training/archive/v1.3.2.tar.gz"
```

Comment:
```suggestion
  url "https://github.com/cloud-annotations/cloud-annotations/archive/v1.3.2.tar.gz"
```
Let's update the `url` too.

Reply:
oops!